File: LIMS/Back/bookmanagement/src/main/java/com/upc/bookmanagement/entity/Comment.java
package com.upc.bookmanagement.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
@TableName("comments")
public class Comment {
    @TableId(value = "comment_id", type = IdType.AUTO)
    private Integer commentId;
    private String title;
    private String username;
    private String comment;
    @TableField("created_at")
    private Date createdAt;
} 

File: LIMS/Front/vue-bookman/src/api/comment.ts
import request from '@/utils/request';

// 获取某本书的评论
export function getCommentsByTitle(title: string) {
  return request.get(`/comments/byTitle?title=${encodeURIComponent(title)}`);
}

// 添加评论
export function addComment(data: { title: string; username: string; comment: string }) {
  return request.post('/comments/add', data);
} 